alcs-frontend/src/app/features/application/review/review.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-review',
+  templateUrl: './review.component.html',
+  styleUrls: ['./review.component.scss'],
+})
+export class ReviewComponent implements OnInit {
```

Comment:
You can remove the implement OnInit if its not doing anything and delete the function